### DISCUSSION TITLE
Limit amount for a cart-item of digital products

### DISCUSSION THREAD

**Discussion by wautersj** [duncanmcclean/simple-commerce#998] [Feature Requests]
### Description

Right now, I can add a single Digital product 5 times, to my cart, or just set the amount to 5 and add it to the cart.
Resulting in the total prince to multiply x 5 (of course)
But receiving a downloads email with only 1 download. (which is quite understandable, in the context of a download).

Lets' view that behaviour from the eyes of a customer. He doesn't understand.
Also; when he accidentally added it to the cart twice, and didn't look to verify before payment; the shop owner has to do admin work when the customer eventually figures it out later on.

This might be countered by limiting the amount for a single digital product in a cart?

![Screenshot 2023-11-20 at 10 37 18](https://github.com/duncanmcclean/sc-digital-products/assets/4070167/d9a5ab33-7299-4d59-b68a-6537b5de9c28)

<img width="604" alt="Screenshot 2023-11-20 at 10 29 40" src="https://github.com/duncanmcclean/sc-digital-products/assets/4070167/47993b5f-0085-4bc6-9cd7-b2d5b89edeaf">

<img width="611" alt="Screenshot 2023-11-20 at 10 29 50" src="https://github.com/duncanmcclean/sc-digital-products/assets/4070167/d9fcdb06-1ed6-4c9e-a209-6026c60357ae">

**Comment by duncanmcclean** [comment]
Just to confirm I'm understanding correctly... you're wanting a way to specify the max number of items a product can be added to the cart in a single order?
